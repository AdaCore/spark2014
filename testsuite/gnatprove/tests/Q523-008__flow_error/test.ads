pragma SPARK_Mode;

with Global;

package Test
is
   function Test return Integer
   is (Global.Val)
     with Global => null;

end Test;
