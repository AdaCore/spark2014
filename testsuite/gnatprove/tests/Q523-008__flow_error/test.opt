ALL XFAIL flow error not reported
