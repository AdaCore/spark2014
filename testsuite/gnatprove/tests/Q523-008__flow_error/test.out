test.ads:8:14: high: "Val" must be listed in the Global aspect of "Test" (SPARK RM 6.1.4(14))
test.ads:9:11: "Val" is referenced in expression function but missing from the Global
test.ads:9:11: "Val" is referenced in expression function but missing from the Global
gnatprove: error during flow analysis and proof
